### PR TITLE
spotLight generic card 

### PR DESCRIPTION
**Pull request by farhanp1502**

### CODE REVIEW

File: src/app/shared/spotlight-card/spotlight-card.component.html
Comment (Vinod-V3) [inline src/app/shared/spotlight-card/spotlight-card.component.html:1]: Remove all the commented lines from the page

File: src/app/shared/spotlight-card/spotlight-card.component.scss
Comment (Vinod-V3) [inline src/app/shared/spotlight-card/spotlight-card.component.scss:1]: Why are the class names given as testimonials and all?

File: src/app/shared/spotlight-card/spotlight-card.component.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-spotlight-card',
+  templateUrl: './spotlight-card.component.html',
+  styleUrls: ['./spotlight-card.component.scss'],
+})
+export class SpotlightCardComponent  {
+  @Input() spotlightstory:any;
```
Comment (Vinod-V3) [inline src/app/shared/spotlight-card/spotlight-card.component.ts:9]: Use camelCase naming convention

File: src/app/shared/spotlight-card/spotlight-card.component.ts
Comment (Vinod-V3) [inline src/app/shared/spotlight-card/spotlight-card.component.ts:1]: Use camelCase naming convention for all the keys used

File: src/app/shared/spotlight-card/spotlight-card.component.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  @Input() index:any;
+  @Output() emitstory = new EventEmitter<any>();
+  constructor() {  }
+
+  emitStory(storyid:any){
```
Comment (Vinod-V3) [inline src/app/shared/spotlight-card/spotlight-card.component.ts:14]: Do we need to emit the event?
Can't we write the redirection logic within this component itself?

File: src/app/shared/spotlight-card/spotlight-card.component.html [hunk 3]
```diff
@@ -0,0 +1,200 @@
+<div class="cards"
```
Comment (Vinod-V3) [inline src/app/shared/spotlight-card/spotlight-card.component.html:1]: Cant we use <mat-card> here?

File: src/environments/environment.ts
Comment (5Amogh) [inline src/environments/environment.ts:1]: Remove the environment ts file

**Review by 5Amogh — CHANGES_REQUESTED** [review]

File: src/app/shared/spotlight-card/spotlight-card.component.scss
Comment (farhanp1502) [reply src/app/shared/spotlight-card/spotlight-card.component.scss:1]: changed the testimonials to spotlight name

File: src/app/shared/spotlight-card/spotlight-card.component.ts
Comment (farhanp1502) [reply src/app/shared/spotlight-card/spotlight-card.component.ts:1]: yea changed

File: src/app/shared/spotlight-card/spotlight-card.component.ts (hunk 2 above)
Comment (farhanp1502) [reply src/app/shared/spotlight-card/spotlight-card.component.ts:14]: we can redirect and the logic changed

File: src/app/shared/spotlight-card/spotlight-card.component.html (hunk 3 above)
Comment (farhanp1502) [reply src/app/shared/spotlight-card/spotlight-card.component.html:1]: the outline margin will be effected

File: src/environments/environment.ts
Comment (farhanp1502) [reply src/environments/environment.ts:1]: removed the environment.ts file changes

**Review by Vinod-V3 — APPROVED** [review]

**Pull request merged**